bot/utils/infractions.py

```diff
@@ -1,3 +1,4 @@
+# Infractions
```

Comment:
```suggestion
```
This line is not acceptable, there is no reason to have a comment saying `# Infractions`